src/domain/interfaces/email.datasource.interface.ts

```diff
@@ -1,6 +1,17 @@
 export const IEmailDataSource = Symbol('IEmailDataSource');
 
 export interface IEmailDataSource {
+  /**
+   * Gets the array of verified emails associated with a Safe address.
```

Comment:
nitpick: "Gets the verified emails associated with a Safe address"

Reply:
Changed in [f1ff610](https://github.com/safe-global/safe-client-gateway/pull/877/commits/f1ff61004d4b054ac07ac7b5750cec600b9d5d8a)